### PR TITLE
In 131918563 edit save functionality to screening people

### PR DESCRIPTION
**Pull request by sivaOSI**

### CODE REVIEW

**Comment by brunoOSI** [comment]
@sivaOSI I don't see the swagger specification for the new endpoint. That should be the starting point to begin working on api modifications.

File: app/controllers/api/v1/participants_controller.rb [hunk 1]
```diff
@@ -10,6 +10,21 @@ def create
         render json: participant, status: :created
       end
 
+      def show
```
Comment (NateOSI) [inline app/controllers/api/v1/participants_controller.rb]: is #show required

File: config/routes.rb [hunk 2]
```diff
@@ -3,7 +3,7 @@
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
   namespace :api do
     namespace :v1 do
-      resources :participants, only: [:create, :destroy]
+      resources :participants, only: [:create, :destroy, :update, :show]
```
Comment (NateOSI) [inline config/routes.rb]: is show required?

File: spec/requests/api/v1/participants_spec.rb [hunk 3]
```diff
@@ -31,57 +31,118 @@
           }
         ]
       }
+    end
+    context 'a participant does not already exist' do
+      it 'creates a participant' do
```
Comment (NateOSI) [inline spec/requests/api/v1/participants_spec.rb]: what creates a participant?

File: spec/requests/api/v1/participants_spec.rb [hunk 4]
```diff
@@ -31,57 +31,118 @@
...
+              street_address: '321 real st',
+              city: 'Real City',
+              state: 'NY',
+              zip: '10010',
+              type: 'Placement'
+            }
+          ]
+        }
+
+        put "/api/v1/participants/#{participant.id}", params: updated_params
```
Comment (NateOSI) [inline spec/requests/api/v1/participants_spec.rb]: what upadtes a participant?

**Pull request merged**